### ISSUE TITLE
Inaccurate version bounds / compile failure

### ISSUE THREAD

**Issue by hvr** [haskell-foundation/foundation#182]
This happens w/ e.g. GHC 7.4.2

```
Configuring component lib from foundation-0.0.3...
Preprocessing library foundation-0.0.3...

Foundation/String/UTF8.hs:62:18:
    Could not find module `GHC.Char'
    Perhaps you meant GHC.MVar (from base)
    Use -v to see a list of the files searched for.

```

**Comment by vincenthz** [comment]
not a supported compiler in 0.0.3

**Comment by hvr** [comment]
@vincenthz then I suggest to simply set the version bounds in `foundation.cabal` accordingly and then we won't have to report this again once foundation-0.0.4 gets uploaded

**Comment by snoyberg** [comment]
Herbert: I was under the impression that you had agreed with Alan to not
file these issues against package authors who have stated they do not agree
with PVP bounds. Was this not the case? If that agreement hasn't been
reached, is there some way package authors can opt out of these reports? It
seems like an unfortunate waste of both your time and the author's to go
through this discussion for each upload.

**Comment by hvr** [comment]
for the record; turns out this was a duplicate of #170 which was fixed rightaway by @NicolasDP via #171   but apparently didn't make it into foundation-0.0.3; so this particular issue shouldn't occur w/ foundation-0.0.4
